### ISSUE TITLE
Upgrade to Bootstrap 4

### ISSUE THREAD

**Issue by daN4cat** [jekkos/opensourcepos#841]
Bootswatch will soon or later upgrade to Bootstrap 4.

We need to track progress here: https://github.com/thomaspark/bootswatch/issues/499

**Comment by erastus** [comment]
now available for Bootstrap 4
https://github.com/thomaspark/bootswatch/releases/tag/v4.1.3

**Comment by daN4cat** [comment]
Yes I know it's available but the work required is too much, also including all the widgets that need to be b4 too.

**Comment by jekkos** [comment]
According to the docs, the bootstrap dom elements should be backwards compatible, so I guess it's a matter of bumping dependencies and then going over all components to see if things are fine
